sync/trypush.py

```diff
@@ -297,6 +297,14 @@ def for_taskgroup(cls, git_gecko, taskgroup_id, sync_type="*", status="open"):
     def treeherder_url(try_rev):
         return "https://treeherder.mozilla.org/#/jobs?repo=try&revision=%s" % try_rev
 
+    def __eq__(self, other):
+        if not hasattr(other, "_ref") and hasattr(other._ref, "_process_name"):
```

Comment:
You mean `not (hasattr(other, "_ref") and hasattr(other._ref, "_process_name"))`?